xline-client/src/types/kv.rs

```diff
@@ -368,7 +368,7 @@ impl From<RangeRequest> for xlineapi::RangeRequest {
 }
 
 /// Request type for `DeleteRange`
-#[derive(Debug)]
+#[derive(Debug, PartialEq)]
```

Comment:
What is this `PartialEq` for?

Reply:
It used in test in xlinectl, it doesn't seems to working to use cfg_attr for conditional comilation for test cfg only .